cmd/bucket-handlers.go

```diff
@@ -447,6 +447,15 @@ func (api objectAPIHandlers) PostPolicyBucketHandler(w http.ResponseWriter, r *h
 		writeErrorResponse(w, ErrMissingContentLength, r.URL)
 		return
 	}
+	resource, err := getResource(r.URL.Path, r.Host, globalDomainName)
+	if err != nil {
+		writeErrorResponse(w, ErrInvalidRequest, r.URL)
```

Comment:
We should add a return here.. so that we don't move forward.

Reply:
shucks, good catch.